scrapers/AssTraffic.yml

```diff
@@ -0,0 +1,60 @@
+name: "AssTraffic"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - asstraffic.com/movies/
+    scraper: sceneScraper
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - asstraffic.com/models/
+    scraper: performerScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h2
+      Date:
```

Comment:
Date imo is safer like this
```yaml
      Date:
        selector: //div[@class="container"]//span[contains(.,"Added")]/text()
        postProcess:
          - replace:
            - regex: "Added "
              with:
          - parseDate: January 2, 2006
```

Reply:
Makes total sense. I guess this was one of my first scrapers where I just copied the absolute path of the element from my chromium plugin here.